.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,40 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: ci-build
+
+# Trigger the workflow when:
+on:
+  # A push occurs to one of the matched branches.
+  push:
+    branches: [master]
+  # Or when a pull request event occurs for a pull request against one of the
+  # matched branches.
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    # NOTE: This name appears in GitHub's Checks API.
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Node.js 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+          cache: yarn
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile | tee yarn-install-out
+      - name: Build app
+        run: yarn build | tee yarn-build-out
+      - name: Print yarn install time
+        run: |
+          echo '```sh' >> $GITHUB_STEP_SUMMARY
+          cat yarn-install-out | tee -a $GITHUB_STEP_SUMMARY
+          echo '```' >> $GITHUB_STEP_SUMMARY
+      - name: Print bundle size
+        run: |
+          echo '```sh' >> $GITHUB_STEP_SUMMARY
+          cat yarn-build-out | tee -a $GITHUB_STEP_SUMMARY
+          echo '```' >> $GITHUB_STEP_SUMMARY
```

Comment:
Let's remove these. I didn't come up with any very useful usecase.
(added in https://github.com/oasisprotocol/explorer/pull/96)